trace/data.go

```diff
@@ -24,21 +26,21 @@ type traceData struct {
 	ApplicationTags           map[string]interface{} `json:"applicationTags"`
 
 	//Trace fields
-	RootSpanId      string                 `json:"rootSpanId"`
+	RootSpanID      string                 `json:"rootSpanId"`
 	StartTimestamp  int64                  `json:"startTimestamp"`
 	FinishTimestamp int64                  `json:"finishTimestamp"`
 	Duration        int64                  `json:"duration"`
 	Tags            map[string]interface{} `json:"tags"`
 }
 
-func (tr *trace) prepareTraceData(ctx context.Context, request json.RawMessage, response interface{}) traceData {
+func (tr *trace) prepareTraceDataModel(ctx context.Context, request json.RawMessage, response interface{}) traceDataModel {
 	tags := tr.prepareTraceTags(ctx, request, response)
-	return traceData{
-		Id:                        plugin.TraceId,
+	return traceDataModel{
```

Comment:
The other plugins, e.g. metric has metricData struct in it. I think it would be better if we also convert them to metricDataModel for consistency.